egs2/iwslt22_dialect/st1/conf/decode_st_conformer.yaml

```diff
@@ -0,0 +1 @@
+decode_pen/pen2.yaml
```

Comment:
what's this?

Reply:
oops it was missing. thanks for catching it